.changeset/gold-bees-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+"@accelint/design-system": minor
```

Comment:
Nit: your changelog message says `patch` but you marked this as a `minor`

Reply:
i've actually been a little confused by which language belongs with which change. so a `patch` is a fix but a feature would have a `minor` (or major) label?

Reply:
A little more nuanced but realistically the loose definitions provided [here](https://docs.npmjs.com/about-semantic-versioning) can be used as a good rule of thumb.